src/test/groovy/org/gradlex/buildparameters/GradleCrossVersionTest.groovy

```diff
@@ -99,7 +99,7 @@ class GradleCrossVersionTest extends Specification {
                 .build()
 
         where:
-        gradleVersion << ["7.1", "7.1.1", "7.2", "7.3.3", "7.4.2", "7.5.1"]
+        gradleVersion << ["7.1", "7.1.1", "7.2", "7.3.3", "7.4.2", "7.5.1", "7.6"]
```

Comment:
It would probably be enough to test agains the first compatible (`7.1.1`) and the last 7.x release (`7.6`).